.github/workflows/functional-test.yaml

```diff
@@ -380,7 +386,9 @@ jobs:
       - name: Login to Azure
         uses: azure/login@v1
```

Comment:
V2 was also released last month which adds Node 20 support and fixes some error messages.